other/refresh-env-var-in-pod/.chainsaw-test/chainsaw-step-04-apply-1.yaml

```diff
@@ -5,3 +5,5 @@ kind: Secret
 metadata:
   name: env-secret
   namespace: refresh-env-var-ns
+  labels:
```

Comment:
Do we need to add this ?

Reply:
Yes. I found it was failing because when Chainsaw would apply this, it removed the labels on the existing Secret which caused no refresh to occur. I assumed this was because you changed the patch/update model a while back.

Reply:
Actually i didn't change the way we patch 🤔 
I'll dig into it, thanks for reporting.

Reply:
Is it better to have an issue logged in Chainsaw for this in that case?